uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/AbsenceType.php

```diff
@@ -132,6 +132,11 @@ private function addBasicDetailsFields() {
       $this->getDAOFieldAttributes('color'),
       TRUE
     );
+    $this->add(
+      'checkbox',
+      'hide_label',
+      ts('Hide leave type label on public calendars and feeds?')
+    );
```

Comment:
I think it makes more sense to add the field after the existing ones (i.e. after "Enabled"). The reason is to not change the of the existing fields